.env.testing

```
@@ -0,0 +1,8 @@
+APP_KEY=
+APP_ENV=testing
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_ENV key should go before the APP_KEY key

(UnorderedKey)